airflow-ctl/src/airflowctl/ctl/cli_config.py

```diff
@@ -172,6 +172,17 @@ def __call__(self, parser, namespace, values, option_string=None):
     help="File path to read from or write to. "
     "For import commands, it is a file to read from. For export commands, it is a file to write to.",
 )
+ARG_OUTPUT = Arg(
+    (
+        "--output",
+        "-o",
+    ),
+    help="Output format. Allowed values: json, yaml, plain, table (default: table)",
```

Comment:
Default is mis-matching to the default 4 lines below

Reply:
Aa thanks! Indeed, I made it json as a default :)